### ISSUE TITLE
Team Inbox Gmail integration UI improvements

### ISSUE THREAD

**Issue by Batchimegb** [erxes/erxes#1116]

**Comment by sergeychikanov** [comment]
@Batchimegb - I am curious what this task includes. I have a few feedback items on the inbox which I was planning to look at/implement in the coming month. I am happy to submit PRs but I was curious what is already planned here. I joined the slack channel so feel free to message any questions there too.

- Browser notifications. In an old inbox project which I am deprecating we used the standard `Notification` API without the persistence flag from Chrome and that worked well everywhere. Are you planning to include browser notifications?
- The default view of the Inbox conversation list is basically "All new + open conversations". This does not change as conversations get assigned or participated in. It would be helpful to show in the conversation list the avatar or name of the assignee / participant in each conversation.
- You are already storing the online state of customers for integrations where it is possible. It would be helpful to show a "green" indication in the conversation list to show the state before the full conversation is loaded for all conversations in the list.
- Currently conversations are marked as unread on a per-user basis. Normally in a ticketing/inbox system conversations which are assigned to another user are not marked as unread because the content might not relevant to the logged in user. Unread conversations total count should be the sum of "new conversations + unassigned & unread by the logged in user" conversations. The only exception would be @mentions.
  - This would also allow a separate filter on "unread" which would be very helpful for auditor/management users.
- The current situation with filters is slightly unclear. When changing from "unassigned" to "resolved", the filter also changes the status but this is not indicated to the user. As a result it not possible to view some filter combinations, for example "Participating + Resolved" or "Assigned to me + New/Open". The default filter value on status should be visible in the inbox.
- Currently there is an idle time per conversation. Are there any plans for an email alert if this idle time is exceeded (eg: takes too long for agents in the inbox to reply to a conversation)?
  - Such an alert should also include the conversation transcript.

@batamar - These are the feedback items I mentioned in the other issue. Thanks a lot for taking a look!
